reference/shipping.v3.yml

```diff
@@ -410,6 +410,16 @@ components:
           example: true
         hs_codes:
           $ref: '#/components/schemas/harmonizedSystemCodes'
+        createdAt:
```

Comment:
While reading the [docs](https://github.com/bigcommerce/api-specs/pull/985/files#diff-8baa1a8874b11300860b8205908bd7256fd88d9ac522bff369b69380c7049f6aR128) it does appear that this will be part of the request body also which we don't want.

Reply:
Indeed. Does this mean we need to separate out request and response? Any thoughts on how best to proceed here @bc-andreadao?

Reply:
I see what you mean. I just separated them out by creating a`customsInformation_request`.